recipe/patches/0017-Unvendor-openssl.patch

```diff
@@ -71,33 +71,33 @@ index f83d4df0d5..b2a9f798f2 100644
      <zlibDir>$(ExternalsDir)\zlib-1.2.11\</zlibDir>
      
 diff --git a/PCbuild/python.vcxproj b/PCbuild/python.vcxproj
-index 8b64e364f1..5b0fa34c36 100644
+index f4d0761..dcf33fe 100644
 --- a/PCbuild/python.vcxproj
 +++ b/PCbuild/python.vcxproj
-@@ -72,6 +72,9 @@
+@@ -88,6 +88,9 @@
    </ItemGroup>
    <ItemGroup>
      <ClCompile Include="..\Programs\python.c" />
-+    <ClCompile Include="$(opensslIncludeDir)\openssl\applink.c">
-+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;$(PreprocessorDefinitions)</PreprocessorDefinitions>
-+    </ClCompile>
++    <ClCompile Include="$(opensslIncludeDir)\openssl\applink.c">

++      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;$(PreprocessorDefinitions)</PreprocessorDefinitions>

++    </ClCompile>


```

Comment:
Line ending artifacts...?

Reply:
The CRLF line endings added in this patch are correct.  This is changing a Windows file which uses these line endings.  The previous patch had incorrect line endings.